canopy/design-system/.gitignore

```diff
@@ -0,0 +1,26 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
.gitignore should only be done at the top-level of the repo (as per our convention).